spec/collector_spec.rb

```diff
@@ -294,6 +294,10 @@
 
   describe '#collect_network_interfaces' do
     before do
+      if not Socket.respond_to?(:getifaddrs) then
```

Comment:
This is usually written as `unless Socket.respond_to?(:getifaddrs)` (then isn't needed at the end of a line)